### PR TITLE
normalize card size

### LINKED ISSUE #152: Make cards the same size
Either use `height: 100%;` or make all banners the same size: card > img { max-height: 6em; content-fit: cover; } can give us some backward compatibility, or we could use hugo to crop the banners to the same standard size.
The cards in the tutorial section are fine: http://localhost:1313/tutorial/

### PR DESCRIPTION
**Pull request by Levrault**
Closes #152 

**Please check if the PR fulfills these requirements:**

- [ x ] The commit message follows our guidelines.
- For bug fixes and features:
    - [ x ] You tested the changes.
    - [ x ] You updated the docs or changelog.

Related issue (if applicable): # https://github.com/GDQuest/website/issues/152

**What kind of change does this PR introduce?**
Normalize card size

**Does this PR introduce a breaking change?**

Some banner doesn't fit the new ratio and some data can be hidden. That with I introduce to class è `card-sm` to prevent breaking change for the new page.

## New feature or change ##

**What is the current behavior?** 

image
image

**What is the new behavior?**
image
image

**Other information**
First PR on project so feel free to give your feedback!

### CODE REVIEW

File: resources/scss/modules/_card.scss [hunk 1]
```diff
@@ -26,10 +27,26 @@
     @include card-outline;
   }
 
+  &.card-sm {
```
Comment (NathanLovato) [inline resources/scss/modules/_card.scss:30]: We're going for [rscss](https://rscss.io/) for the css code style and structure. While it's a progressive refactor, could you name it `.-small`?

**Comment by Levrault** [comment]
Yop I can make the change myself, it will help me understand your standards and gain some experience with hugo. And sorry about the rcss struc I ain't done sass for the past 3 years so it's been a long time

File: layouts/_default/section.html [hunk 2]
```diff
@@ -13,7 +13,8 @@
         {{ if and .Paginator.Pages (or (and (isset .Params "paginate") .Params.paginate) (not (isset .Params "paginate"))) }}
         <section class="grid split2 container list">
             {{ range .Paginator.Pages }}
-            <div>{{ .Render "summary" }}</div>
+            {{ .Scratch.Set "classname" "card-sm" }}
```
[the remark this answers is not in the document]
Comment (Levrault) [reply layouts/_default/section.html:16]: Strangly even if the hugo version should support it (has seen in the doc https://gohugo.io/templates/introduction/#variables), those variable seems to only work for the current template. They can't be read inside summary.html :$

**Comment by NathanLovato** [comment]
I see the height: 100% is causing some issue. There's a shortcode, calltoaction, that breaks:

Screenshot from 2020-05-29 14-13-27

**Comment by NathanLovato** [comment]
Here's how it currently looks:

Screenshot from 2020-05-29 14-15-40

**Comment by Levrault** [comment]
Alright problem happen only on firefox browser (work fine on chrome and Internet Explorer)

**Comment by Levrault** [comment]
After some investigations, Firefox seems to have problem computing the height when the content is not self contained inside a div. Should be good now if I ain't forgot anything.

image

Tested on Internet Explorer, Edge, Firefox and chrome

**Comment by NathanLovato** [comment]
Thanks much!

**Pull request merged**